lib/tab-smart-sort.coffee

```diff
@@ -56,14 +56,14 @@ getSortStr = (item) ->
   if not caseSensitive then sortStr = sortStr.toLowerCase()
   sortStr
 
-addItem = (newItem) ->
+addItem = (newItem, options) ->
   if newItem in @items then return
   newSortStr = getSortStr newItem
   lastSortStr = ''
   for item, newIndex in @items
     sortStr = getSortStr item
     if lastSortStr <= newSortStr < sortStr then break
     lastSortStr = sortStr
-  originalAddItem.call @, newItem, newIndex
+  originalAddItem.call @, {index: newIndex, pending: options.pending}

```

Comment:
This doesn't actually work. You need to pass newItem as well, and you should not be rebuilding the options hash.